docs/tutorials/13_quantum_random_access_optimizer.ipynb

```diff
@@ -0,0 +1,892 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "maxcut_partition = maxcut.interpret(results)\n",
+    "print(\n",
+    "    f\"The obtained solution places a partition between nodes {maxcut_partition[0]} \"\n",
+    "    f\"and nodes {maxcut_partition[1]}.\"\n",
+    ")\n",
+    "maxcut.draw(results, pos=nx.spring_layout(graph, seed=seed))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Inspect the results of subroutines"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `MinimumEigensolverResult` ([details](https://qiskit.org/documentation/stubs/qiskit.algorithms.MinimumEigensolverResult.html)) that results from performing VQE on the relaxed Hamiltonian is available:"
```

Comment:
```suggestion
    "The [MinimumEigensolverResult](https://qiskit.org/ecosystem/algorithms/stubs/qiskit_algorithms.MinimumEigensolverResult.html) that results from performing VQE on the relaxed Hamiltonian is available:"
```
This needs to be the link to the qiskit_algorithms API - I updated it and also altered to removed the details a in brackets as the link - why not just have it be the link.